.github/workflows/pr-build.yml

```diff
@@ -53,13 +46,9 @@ jobs:
           distribution: adopt
           java-version: 11
       - uses: burrunan/gradle-cache-action@v1
-        name: ./gradlew integrationTest
-        id: integration-test
-        timeout-minutes: 10
+        name: Integration Tests
         with:
-          arguments: --stacktrace --max-workers 2 integrationTest
```

Comment:
Because we're setting the parallel gradle property I found not limiting the max workers to overrun the actions runner and cause flake.  I'm not sure how many cores are advertised but in circle ci the host machine's are instead of what's granted to the executing container.

Reply:
We should get out of circle and use GHA, correct?

Reply:
This has been done (https://github.com/open-telemetry/opentelemetry-java-contrib/issues/30) and I'm referring to an issue I observed in actions.

Reply:
We haven't had much issue even in our huge instrumentation repo build :P I've looked at some build scans and it seems to be using about 2 workers by default on GHA so looks like that issue was probably fixed by now